worker_node/job/send.js

```diff
@@ -129,7 +142,7 @@ class Send extends Base {
   }
 
   async sendTxs() {
-    const delayTime = this.config.duration / this.config.numberOfTransactions * 1000;
+    const delayTime = (this.config.duration / this.config.numberOfTransactions * 1000) * 0.97;
```

Comment:
Could you define 0.97 as a constant variable for readability?

Reply:
Done!